bin/check-sqs-messages.rb

```diff
@@ -96,14 +102,47 @@ def aws_config
   def run
     AWS.config aws_config
     sqs = AWS::SQS.new
-    messages = sqs.queues.named(config[:queue]).approximate_number_of_messages
+    if config[:prefix] == ''
+      if config[:queue] == ''
```

Comment:
Sorta -- then I'd have to redo `if config[:prefix] == ""` again immediately after.